pages/v2/signUp.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Box,
+  Flex,
+  Stack,
+  Heading,
+  Text,
+  Container,
+  Input,
+  Button,
+  SimpleGrid,
+  Avatar,
+  AvatarGroup,
+  useBreakpointValue,
+  IconProps,
+  Icon,
+  InputGroup,
+  InputLeftAddon,
+  FormControl,
+  FormLabel,
+  Select,
+  FormHelperText,
+  VStack,
+  HStack,
+} from '@chakra-ui/react';
+
+const colleges = [
+  { id: 1, name: 'University of Illinois at Urbana-Champaign' },
+  { id: 2, name: 'University of Illinois at Chicago' },
+  { id: 3, name: 'University of Illinois at Springfield' },
+  { id: 4, name: 'St. Louis University' },
+  { id: 5, name: 'Massachusetts Institute of Technology' },
+];
+
+export default function JoinOurTeam() {
+  const BPL = useBreakpointValue({ base: 'md', md: 'lg' });
+  return (
+    <HStack
+      bg={'gray.50'}
+      rounded={'xl'}
+      p={{ base: 4, sm: 6, md: 8 }}
+      spacing={{ base: 8 }}
+      alignItems={'start'}
+      justifyContent="space-around"
+      // h="100vh"
+      // overflow="hidden"
+    >
+      <Stack maxW="30%" spacing={4}>
+        <Heading
+          color={'gray.800'}
+          lineHeight={1.1}
+          fontSize={{ base: '2xl', sm: '3xl', md: '4xl' }}
+        >
+          Become a voyager / crew member
+          <Text
+            as={'span'}
+            bgGradient="linear(to-r, red.400,pink.400)"
+            bgClip="text"
+          >
+            !
+          </Text>
+        </Heading>
+        <Text color={'gray.500'} fontSize={{ base: 'sm', sm: 'md' }}>
+          We’re looking for amazing crew-members just like you! Become a part of
+          our rockstar team and help make an impact!
+        </Text>
+      </Stack>
+      <Box as={'form'} height="80vh" overflow={'scroll'} mt={10}>
+        <Stack spacing={4}>
+          <InputGroup>
+            <InputLeftAddon w="100px"> Name </InputLeftAddon>
+            <Input placeholder="John Doe" />
+          </InputGroup>
+          <InputGroup>
+            <InputLeftAddon w="100px">Email</InputLeftAddon>
+            <Input placeholder="john@doe.com" />
+          </InputGroup>
+          <Input placeholder="Name" bg={'gray.100'} color={'gray.500'} />
+          <FormControl isRequired>
+            <FormLabel htmlFor="name"> Name</FormLabel>
```

Comment:
Why the whitespace before `Name`. Make sure you keep the code clean, I don't want the code reviews to be about this kind of things from the next time. You need to make sure you work out these kinds of things by yourself.

Reply:
I'll make sure to keep it clean!